torchci/rockset/prodVersions.json

```diff
@@ -1,8 +1,20 @@
 {
-  "hud_query": "86804e2dcb570a13",
-  "commit_jobs_query": "8eba563cd0fb72a7",
-  "flaky_test_query": "482db17169272025",
-  "original_pr_hud_query": "a07ff9976e0363e8",
-  "issue_query": "f29a1afe94563601",
-  "failure_samples_query": "2961636418a148c2"
+  "commons": {
+    "hud_query": "86804e2dcb570a13",
+    "commit_jobs_query": "8eba563cd0fb72a7",
+    "flaky_test_query": "482db17169272025",
+    "original_pr_hud_query": "a07ff9976e0363e8",
+    "issue_query": "f29a1afe94563601",
+    "failure_samples_query": "2961636418a148c2"
+  },
+  "pytorch_dev_infra_kpis": {
+    "num_reverts": "latest",
```

Comment:
I guess I missed this before. We should always use exact hashes, not `latest`, which can change for whatever reason.

Reply:
can do it after though